randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -163,7 +178,7 @@ Extra - asset_id: 4
   * Layers: default
   * Normal Door to Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
-      Grapple Beam or Movement (Beginner)
+      Grapple Beam or Space Jump or Movement (Intermediate)
```

Comment:
What about WJ/IBJ/HJ?

Reply:
You're thinking about the upper misisle tank. This is the one on the sea floor. You only need morph to get it, and the knowledge is there so that the player knows it doesn't need gravity and that you have to shoot a block off screen to get out.

Reply:
This is not a missile tank tho...? This is left door (moat) -> right door (wrecked ship entrance)

Reply:
I think your comments might get misaligned in VSCode. I'll look at it again later. I might stick to doing reviews only from the Github website. Thanks for the clarity.